docs_src/src/pages/documentation/api_reference/multithreading.mdx

```diff
@@ -0,0 +1,61 @@
+export const description =
+  'On this page, we’ll dive into the different conversation endpoints you can use to manage conversations programmatically.'
+
+## Multithreading
+
+Batman wondered about the nuances of using multithreading in Robyn.
+
+
+<Row>
+<Col>
+Whilst using multithreading in Robyn, the global variables are not protected. One should use `multiprocessing.Value` for achieving the required result.
+</Col>
+  <Col sticky>
+
+    <CodeGroup title="Request" tag="GET" label="/hello_world">
+
+    ```python {{ title: 'untyped' }}  
+        import threading
+        import time
+        from multiprocessing import Value
+
+        from robyn import Robyn, Request
+
+        app = Robyn(__file__)
+
+        count = Value("i", 0)
```

Comment:
this is required since the count is shared across multiple processes. And we need the value to protect it within the process, while accessing the count from two different threads.